### ISSUE TITLE
[Feature] Functions Documentation on Website lacks Version Information

### ISSUE THREAD

**Issue by Howard0101** [PSAppDeployToolkit/PSAppDeployToolkit#1042]
### Summary of the new feature / enhancement

I suggest that a version information is added in the documentation of the Functions on the website. I know that the most recent release will have all the functions described. It is just that I do need to "repair" or tweak a little with every new release on the nagging missing features like #1001 and #344 and #41 and so on... This takes time and the releases keep coming. So I end up with customers still using an old template from me. So it would be nice to know what function was available since what version of PS AppDeploy Toolkit. For example: Function Copy-FileToUserProfiles exists since V3.10.x

### Proposed technical implementation details (optional)

In the description of the Functions under the Header Notes add the following:
NOTES
added since V3.10.1

**Comment by mjr4077au** [comment]
Hey there! We could do this, but it should be noted that no release of PSADT is considered LTS, etc. Only the latest release is supported.

**Comment by Howard0101** [comment]
Yes, Thank you. I am aware that there is no LTS or something like that. I would really appreciate if you are willing to still implement this. Thanks again.

**Comment by mjr4077au** [comment]
This has been addressed 🥳
